src/linear_algebra/clifford_algebra/basic.lean

```diff
@@ -193,13 +193,26 @@ begin
   exact alg_hom.congr_fun of_id a,
 end
 
+--set_option trace.simplify.rewrite true
 /-- A Clifford algebra with a zero quadratic form is isomorphic to an `exterior_algebra` -/
 def as_exterior : clifford_algebra (0 : quadratic_form R M) ≃ₐ[R] exterior_algebra R M :=
 alg_equiv.of_alg_hom
-  (clifford_algebra.lift 0 ⟨(exterior_algebra.ι R), by simp⟩)
-  (exterior_algebra.lift R ⟨(ι (0 : quadratic_form R M)), by simp⟩)
-  (exterior_algebra.hom_ext $ linear_map.ext $ by simp)
-  (clifford_algebra.hom_ext $ linear_map.ext $ by simp)
+  (clifford_algebra.lift 0 ⟨(exterior_algebra.ι R),
+    by simp only [forall_const, ring_hom.map_zero,
+                  exterior_algebra.ι_sq_zero, quadratic_form.zero_apply]⟩)
+  (exterior_algebra.lift R ⟨(ι (0 : quadratic_form R M)),
+    by simp only [forall_const, ring_hom.map_zero,
+                  quadratic_form.zero_apply, ι_sq_scalar]⟩)
+  (exterior_algebra.hom_ext $ linear_map.ext $
+    by simp only [alg_hom.comp_to_linear_map, linear_map.coe_comp,
+                  function.comp_app, alg_hom.to_linear_map_apply,
+                  exterior_algebra.lift_ι_apply, clifford_algebra.lift_ι_apply,
+                  alg_hom.to_linear_map_id, linear_map.id_comp, eq_self_iff_true, forall_const])
+  (clifford_algebra.hom_ext $ linear_map.ext $
+    by simp only [alg_hom.comp_to_linear_map, linear_map.coe_comp,
+                  function.comp_app, alg_hom.to_linear_map_apply,
+                  clifford_algebra.lift_ι_apply, exterior_algebra.lift_ι_apply,
+                  alg_hom.to_linear_map_id, linear_map.id_comp, eq_self_iff_true, forall_const])
```

Comment:
I assume you changed this because it was too slow? Is it any better if you use `dsimp, simp`?

Reply:
This was one of the already slow proofs that turned into an actual timeout in this PR. I don't think it's worth it to play further with it, since the timeout only showed up on CI and not locally so even if it works it'll take hours to find out.

Reply:
If the timeout is only showing up in CI it may be because you are using different settings locally. I think CI runs with `-T 100000`.

Reply:
(Not saying you should do anything here, however.)